### PR TITLE
2023-08-25

### PR DESCRIPTION
**Pull request by allisonmacmillan**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.beeswax</groupId>
   <artifactId>beeswax-api</artifactId>
-  <version>2021-02-05</version>
+  <version>2023-08-25</version>
```
Comment (akshadavkulkarni) [inline pom.xml:6]: Is this change only for Display purpose or does it trigger any build/package pipelines?

Comment (allisonmacmillan) [reply pom.xml:6]: It's part of the process outlined in [this wiki](https://wiki.freewheel.tv/display/BWEN/How+To+update+the+public+Beeswax+API+repo). We've previously made some changes in this repo but haven't actually released it for a few years.

Comment (allisonmacmillan) [reply pom.xml:6]: To answer your question - no it doesn't trigger any pipelines but I'll use it to deploy to maven.

**Review by akshadavkulkarni — APPROVED** [review]
LGTM. Wiki mentioned updating README [here](https://wiki.freewheel.tv/display/BWEN/How+To+update+the+public+Beeswax+API+repo#:~:text=Then%20update%20the%20version%20number%20in%20README.md%20in%20the%20same%20way) too but not sure since it doesnt contain version number.

**Comment by akshadavkulkarni** [comment]
Maybe useful to update the PR title to mention the change

**Pull request merged**